datahub-graphql-core/src/main/resources/auth.graphql

```diff
@@ -26,7 +44,33 @@ input GetAccessTokenInput {
   The duration for which the Access Token is valid.
   """
   duration: AccessTokenDuration!
+}
+
+input CreateAccessTokenInput {
+  """
+  The type of the Access Token.
+  """
+  type: AccessTokenType!
+
+  """
+  The actor associated with the Access Token.
+  """
+  actorUrn: String!
+
+  """
+  The duration for which the Access Token is valid.
+  """
+  duration: AccessTokenDuration!
+
+  """
+  The name of the token to be generated.
+  """
+  name: String!
```

Comment:
Nice the name is required! 

This means you don't need to do an explicit null check in the resolver. This can be a followup